web-bluetooth/read-characteristic-value-changed.html

```diff
@@ -0,0 +1,55 @@
+---
+feature_name: Web Bluetooth / Read Characteristic Value Changed
+chrome_version: 48
+check_min_version: true
+feature_id: 5264933985976320
+icon_url: icon.png
+---
+
+{% include_relative _includes/intro.html %}
+
+<p>This sample illustrates the use of the Web Bluetooth API to read battery
+level and be notified of changes from a nearby Bluetooth Device advertising
+Battery information with Bluetooth Low Energy. You may want
+to try this demo with the BLE Peripheral
+Simulator App from the <a target="_blank"
+href="https://play.google.com/store/apps/details?id=io.github.webbluetoothcg.bletestperipheral">Google
+Play Store</a>. Note that a <a href="battery-level.html">simpler demo for retrieving battery level
+information</a> is also available.</p>
```

Comment:
Looking at this sample, it is only slightly more than the existing [notifications]( https://googlechrome.github.io/samples/web-bluetooth/notifications.html) one. What value does this sample add beyond that one? Describe that.